airbyte-cdk/python/airbyte_cdk/sources/streams/http/http_client.py

```diff
@@ -208,9 +285,14 @@ def _send(self, request: requests.PreparedRequest, request_kwargs: Mapping[str,
 
         # TODO: Consider dynamic retry count depending on subsequent error codes
```

Comment:
Is this done as part of this PR?

Reply:
No, I'll make a follow-up issue for this though.